src/links/v1/providers.json

```
@@ -205,6 +205,17 @@
         "homepage": "https://www.crystallography.net/tcod",
         "link_type": "external"
       }
+    },
+    {
+      "type": "links",
+      "id": "2dmatpedia",
```

```suggestion
      "id": "twodmatpedia",
```
This represents the provider prefix as well :)

---

what about the folder name, does it also need to be `twodmatpedia` then?

---

Hi @migueldiascosta, that might also make sense! Good to see that we have our first edge case in not allowing numbers in the prefix...

Also thanks for making this implementation with our tools! As @CasperWA suggested we are more than happy to meet to help streamline the process and make the package more ergonomic.